posthog/api/error_tracking.py

```diff
@@ -55,3 +61,23 @@ class ObjectStorageUnavailable(Exception):
 #                 code="object_storage_required",
 #                 detail="Object storage must be available to allow source map uploads.",
 #             )
+
+
+class ErrorTrackingStackFrameSerializer(serializers.ModelSerializer):
```

Comment:
Non-blocking but, any reason to limit this to only returning the raw_id and context? The whole frame record will be useful going forward, for e.g. symbol set management, I think.

Reply:
This is under the `error_tracking/stack_frames/contexts` endpoint which takes an array of `ids` as params. I was planning on having a more conventional `error_tracking/stack_frames` endpoint that returns the whole frame as part of the symbol set management PR (next on my list)